docs/user/security/securing-communications/index.asciidoc

```diff
@@ -4,127 +4,178 @@
+`server.ssl.keystore.password` to an empty string.
+--
+
++
+For more information about settings for certificates and keys, see <<settings,{kib} configuration settings>>.
 
-For more information, see <<settings,{kib} configuration settings>>.
+. *Configure {kib} to enable TLS for inbound connections.*
++
 --
+Specify that TLS/SSL is used in `kibana.yml`:
+
+[source,yaml]
+--------------------------------------------------------------------------------
+server.ssl.enabled: true
+--------------------------------------------------------------------------------
+--
+
+. *Restart {kib}.*
+
+After making these changes, you must always access {kib} via HTTPS. For example, https://<your_kibana_host>.com.
 
 [[configuring-tls-kib-es]]
 ==== Encrypting traffic between {kib} and {es}
 
-NOTE: To perform this step, you must 
-{ref}/configuring-security.html[enable the {es} {security-features}] or you 
-must have a proxy that provides an HTTPS endpoint for {es}. 
+NOTE: To perform this step, you must {ref}/configuring-security.html[enable the {es} {security-features}] or you must have a proxy that
+provides an HTTPS endpoint for {es}.
+
+. *{ref}/configuring-tls.html#tls-http[Enable TLS on the HTTP layer in {es}].*
 
-. Specify the HTTPS URL in the `elasticsearch.hosts` setting in the {kib}
-configuration file, `kibana.yml`:
+. *Obtain the certificate authority (CA) certificate chain for {es}.*
 +
 --
-[source,yaml]
+{kib} needs the appropriate CA certificate chain to properly establish trust when connecting to your {es} cluster.
+
+If you followed the {es} documentation for {ref}/configuring-tls.html#node-certificates[Generating Node Certificates], then you likely have
+a PKCS#12 file for each your {es} nodes. You can extract the CA certificate chain in PEM format from one of these files like so:
+
+[source,sh]
 --------------------------------------------------------------------------------
-elasticsearch.hosts: ["https://<your_elasticsearch_host>.com:9200"]
+openssl pkcs12 -in elastic-certificates.p12 -cacerts -nokeys -out elasticsearch-ca.crt
 --------------------------------------------------------------------------------
+--
 
-Using the HTTPS protocol results in a default
-`elasticsearch.ssl.verificationMode` option of `full`, which utilizes hostname
-verification.
+. *Configure {kib} to trust the {es} certificates for the HTTP layer.*
++
+You need to specify the information required to access your {es} CA certificate chain.
 
-For more information, see <<settings,{kib} configuration settings>>. 
+.. If your CA certificate chain is in PEM format:
++
+--
+Specify one or more certificates in `kibana.yml`:
+
+[source,yaml]
+--------------------------------------------------------------------------------
+elasticsearch.ssl.certificateAuthorities: ["/path/to/elasticsearch-ca.crt"]
+--------------------------------------------------------------------------------
 --
```

Comment:
If the user followed the instructions above, they will have a PEM-formatted CA cert. Because this is the likely scenario, I moved this sub-section above the one for PKCS#12.